### ISSUE TITLE
cannot open compressed file '/Users/xx.rds', probable reason 'Permission denied

### ISSUE THREAD

**Issue by ctlamb** [ropensci/rtweet#413]
```
> library(rtweet)
> rt <- search_tweets(
+   "#rstats", n = 18000, include_rts = FALSE
+ )
Requesting token on behalf of user...
Waiting for authentication in browser...
Press Esc/Ctrl + C to abort
Authentication complete.
Error in gzfile(file, mode) : cannot open the connection
In addition: Warning message:
In gzfile(file, mode) :
  cannot open compressed file '/Users/clayton1.rds', probable reason 'Permission denied'
```

Hello I am unable to authenticate my token either by the browser as shown above, or with an app and credentials. How do I point the save to somewhere else than "/Users/clayton1.rds' where I clearly don't have permissions

**Comment by MichaelChirico** [comment]
Same thing for me on my first installation / auth. Adding `sessionInfo()`:

```
R version 4.0.0 (2020-04-24)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Mojave 10.14.6

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] rtweet_0.7.0       RColorBrewer_1.1-2 colourvalues_0.3.4 data.table_1.12.9 

loaded via a namespace (and not attached):
 [1] httr_1.4.1     compiler_4.0.0 magrittr_1.5   R6_2.4.1       promises_1.1.0 later_1.0.0    tools_4.0.0   
 [8] curl_4.3       Rcpp_1.0.4.6   jsonlite_1.6.1 httpuv_1.5.2   openssl_1.4.1  askpass_1.1  
```

**Comment by HarryKentTest** [comment]
Hi there, 
I'm having the same issues with authentication. I saw you added a fix, I am quite new to R, would you be able to let me know how I can apply this fix on my side please?

Thanks very much

**Comment by MichaelChirico** [comment]
You can try `remotes::install_github("MichaelChirico/rtweet@fix-path")`. Please be sure to monitor this page & return to a more official installation later on.

**Comment by llrs** [comment]
Should be closed by #417. Closing this now
